docs/development/getting_started.rst

```diff
@@ -1,163 +1,101 @@
-Getting Started
-===============
+Contributing to SecureDrop
+==========================
 
-.. include:: ../includes/docs-branches.txt
+If you have time, one hour or a year, we welcome your help in various
+areas. We are commited to make it as easy as possible for you to
+contribute.
 
-Prerequisites
--------------
+* `Programmers`_, to help us developing SecureDrop.
+* `Devops`_, to work on the community maintained SecureDrop infrastructure.
+* `Release managers`_, to create and maintain Debian GNU/Linux packages and repositories.
+* `Writers`_, to help improve the documentation.
+* `Translators`_, to translate SecureDrop
+* `Moderators & Support`_, to answer on `the forums <https://forum.securedrop.club/c/support>`_.
+* `Designers`_, for SecureDrop source and journalist web interfaces and tails customization.
```

Comment:
Tails is a proper noun (i.e. should be capital)

Reply:
done